homeassistant/components/history_stats/sensor.py

```diff
@@ -163,6 +163,8 @@ def __init__(
         self._process_update()
         if self._type == CONF_TYPE_TIME:
             self._attr_device_class = SensorDeviceClass.DURATION
+            # Show up to second resolution
+            self._attr_suggested_display_precision = 4
```

Comment:
Let's keep it as it, so it doesn't become a breaking change for other people. It had a precision of 2 in the old situation.

```suggestion
            self._attr_suggested_display_precision = 2
```

Reply:
Made suggested change